webcompat/templates/layout.html

```diff
@@ -33,6 +33,7 @@
   <link href="https://webcompat.github.io/design/css/main.css" type="text/css" rel="stylesheet">
 </head>
 <body data-username="{{ session.username }}">
+  {% include "shared/svg-icons.html" %}
```

Comment:
Huh. So we did not shift the contents of the `<body>` tag one level to the right. I have no idea why, and I'd love to change that, but that would mess up the line history... 🙈 

It's probably best to stay at the 0th indentation level for this line, just for consistency.